pipelines/main/platforms/upload_macos.yml

```
@@ -32,7 +32,7 @@ steps:
       #MACOS_CODESIGN_IDENTITY="2053E9292809B66582CA9F042B470C0929340362"
       MACOS_CODESIGN_IDENTITY="-"
       APP_NAME="Julia-$${MAJMIN}.app"
-      DMG_NAME="$$(basename $$JULIA_BINARYDIST_FILENAME8 .tar.gz).dmg"
+      DMG_NAME="$(basename $$JULIA_BINARYDIST_FILENAME .tar.gz).dmg"
```

I suspect that `$(...)` will trip Buildkite up and that we should instead do `$(...)`.

```suggestion
      DMG_NAME="$$(basename $$JULIA_BINARYDIST_FILENAME .tar.gz).dmg"
```

---

I thought the double $ was for external variables, I could be wrong however